backend/apid/middlewares/authorization_attributes.go

```diff
@@ -24,8 +24,8 @@ type AuthorizationAttributes struct{}
 // The general, expected format of a path is one of the following:
 // /apis/{group}/{version}/namespaces, to create or list namespaces
 // /apis/{group}/{version}/namespaces/{name}
-// /apis/{group}/{version}/namespaces/{namespace}/{kind}
-// /apis/{group}/{version}/namespaces/{namespace}/{kind}/{name}
+// /apis/{group}/{version}/namespaces/{namespace}/{type}
```

Comment:
I think we should actually use `resource` here and make a distinction between a type and the resource; For example, the kind is `entity` but the resource here in the REST request should be `entities`.

Reply:
Good point, and to add to it: the related field in the `AuthorizationAttributes` is called `Resource`, so let's unify all of this!

I'll change `type` to `resource` in the routes.